packages/react-dom/src/client/ReactDOMTextarea.js

```diff
@@ -157,7 +157,9 @@ export function postMountWrapper(element: Element, props: Object) {
   // will populate textContent as well.
   // https://developer.microsoft.com/microsoft-edge/platform/issues/101525/
   if (textContent === node._wrapperState.initialValue) {
-    node.value = textContent;
+    if (textContent) {
```

Comment:
(As a note: It seems like `Node.textContent` has to be either a string or `null`: https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Return_value so the loose check seems OK, maybe we still want to have an explicit `textContent !== '' && textContent !== null` though)

Reply:
updated to use more explicit check